src/functionalTest/resources/features/common/users/CrdBeftaUser2.td.json

```diff
@@ -0,0 +1,6 @@
+{
+  "_guid_": "CrdBeftaUser2",
+
+  "username": "[[$TEST_AM_CRD_USER1_BEFTA]]",
+  "password": "[[$TEST_AM_USER2_BEFTA_PWD]]"
```

Comment:
you can remove this user as both have same username(TEST_AM_CRD_USER1_BEFTA).
note: password will be same all befta users. in this case for these two TEST_AM_USER2_BEFTA_PWD and TEST_AM_CRD_USER1_BEFTA_PASSWORD